process/block/metablock.go

```diff
@@ -594,7 +594,12 @@ func (mp *metaProcessor) createMiniBlocks(
 
 	miniBlocks := make(block.Body, 0)
 	if mp.epochStartTrigger.IsEpochStart() {
-		return miniBlocks, nil
+		peerMiniBlocks, err2 := mp.generateValidatorMiniBlocks()
+		if err2 != nil {
+			return nil, err2
+		}
+
+		return peerMiniBlocks, nil
```

Comment:
make a merge from economics v2 to here, as a lot of things has changed, and it will be hard later on.

Reply:
fixed